test/models/database/post/post_data_test.dart

```diff
@@ -23,8 +18,14 @@ void main() {
       );
 
       final actualHash = data.hashCode;
-
       expect(actualHash, expectedHash);
+
+      const geoPoint = GeoPoint(40, 20);
```

Comment:
I believe we might use `test/mocks/data/mock_position.dart/userPosition1` here.
```suggestion
      const geoPoint = userPosition1;
```

Reply:
Well seen!
If I may add, I think it may also apply to all `GeoPoint(40, 20)` of `post_upvote_repository_test.dart` and `post_repository_test.dart` too (if you agree with my reasoning, probably Ctrl+F the whole test folder for `GeoPoint(40, 20)` and `GeoPoint(0, 0)`)

Reply:
Yes, that was definitely the point of creating userPosition1, thanks for catching it!

Reply:
Oh thanks! I remember doing a ctrl+f to find those, but somehow I missed it.
Fixed in 2b8946e972325a8a7222a3408678243b03ff332d